driver/network/privilege.go

```diff
@@ -70,7 +71,16 @@ func (d *Driver) deescalate(currentPriv string) error {
 func (d *Driver) determineCurrentPriv(currentPrompt string) ([]string, error) {
 	var matchingPrivLevels []string
 
+PrivLevel:
 	for privName, privData := range d.PrivilegeLevels {
+		if len(privData.PatternNotContains) > 0 {
```

Comment:
seems like you don't need this check, because the next `range` statement simply will be skipped when given an empty slice

Reply:
Ah derp. Makes sense! Thanks man, will update a bit later today and merge!